README.md

```diff
@@ -9,7 +9,7 @@ Metrics package.
 
 ### Collected Data
 
-* A unique identifier that is generated by computing the [SHA-1][SHA1] of the
+* A unique identifier that is generated by computing the [SHA-256][SHA-256] of the
   machine's [MAC address][MAC].
```

Comment:
The README needs to be updated as well to reflect that the MAC address is no longer used.